packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/RemoteMessageSerializer.java

```diff
@@ -87,7 +87,7 @@ private static Bundle toBundle(@Nullable RemoteMessage.Notification notification
       serializedNotification.putString("notificationPriority", null);
     }
     serializedNotification.putString("sound", notification.getSound());
-    serializedNotification.putBoolean("sticky", notification.getSticky());
+    serializedNotification.putBoolean("autoCancel", !notification.getSticky());
```

Comment:
See [docs](https://firebase.google.com/docs/reference/android/com/google/firebase/messaging/RemoteMessage.Notification#getSticky()) and [here](https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages#androidnotification)

```
When set to false or unset, the notification is automatically dismissed when the user clicks it in the panel. When set to true, the notification persists even when the user clicks it.
```